### PR TITLE
Migrate sdk-extensions to kts.

### PR DESCRIPTION
**Pull request by anuraaga**

### CODE REVIEW

File: sdk-extensions/zpages/README.md [hunk 1]
```diff
@@ -1,34 +1,7 @@
...
-
 This module contains code for the OpenTelemetry Java zPages, which are a collection of dynamic HTML
-web pages embedded in your app that display stats and trace data. Learn more in [this blog post][zPages blog];
-
-<!--- TODO: Update javadoc -->
-[javadoc-image]: https://www.javadoc.io/badge/io.opentelemetry/opentelemetry-sdk-contrib-auto-config.svg
-[javadoc-url]: https://www.javadoc.io/doc/io.opentelemetry/opentelemetry-sdk-contrib-auto-config
-[zPages blog]: https://medium.com/opentelemetry/zpages-in-opentelemetry-2b080a81eb47
-
-## Quickstart
```
Comment (anuraaga) [inline sdk-extensions/zpages/README.md:13]: I don't think we need this for alpha modules, and the above was out of date too. Just went ahead and deleted to make the gradle simpler.

File: sdk-extensions/resources/build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+plugins {
+    `java-library`
+    `maven-publish`
+
+    // TODO(anuraaga): Use reflection for RuntimeMXBean which is not included in Android.
```
Comment (jkwatson) [inline sdk-extensions/resources/build.gradle.kts:5]: we need an issue for this, yes?

Comment (jkwatson) [reply sdk-extensions/resources/build.gradle.kts:5]: https://github.com/open-telemetry/opentelemetry-java/issues/2815

**Review by jkwatson — APPROVED** [review]
Thanks!

File: sdk-extensions/resources/build.gradle.kts (hunk 2 above)
Comment (anuraaga) [reply sdk-extensions/resources/build.gradle.kts:5]: Thanks!

**Comment by jkwatson** [comment]
needs a rebase

**Pull request merged**